torchensemble/_base.py

```diff
@@ -44,10 +44,10 @@ class BaseModule(abc.ABC, nn.Module):
     def __init__(self,
                  estimator,
                  n_estimators,
+                 logger,
```

Comment:
Could we change the location of ``logger`` to the place right below ``estimator_args`` ? I think it's better because ``logger``, ``cuda``, and ``n_jobs`` are all utility-related.

Reply:
It's okay but I can't find out an elegant way to set the default value of `logger`. An default global logger may be created in `utils.py` to be assigned. I'll try to fix it in few days.

Reply:
I think a default logger is preferred, thanks!